File: homeworks/hw3/Models/Rectangle.java
package homeworks.hw3.Models;

public class Rectangle extends Figure {

    private double length;
    private double width;

    public Rectangle(double length, double width) {
        this.length = length;
        this.width = width;
    }

    public double getPerimeter() {
        return 2 * (length + width);
    }

    public double getArea() {
        return length * width;
    }

    public double getLength() {
        return length;
    }

    public double getWidth() {
        return width;
    }

}

File: finaltask/models/rectangle.py
from finaltask.models.figure import Figure


class Rectangle(Figure):

    def __init__(self, length, width):
        self.length = length
        self.width = width

    def get_perimeter(self):
        return 2 * (self.length + self.width)

    def get_area(self):
        return self.length * self.width

    def get_length(self):
        return self.length

    def get_width(self):
        return self.width
